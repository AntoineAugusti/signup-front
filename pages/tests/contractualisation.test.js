import ReactTestRenderer from 'react-test-renderer'

describe('pages | contractualisation', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<contractualisation />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
