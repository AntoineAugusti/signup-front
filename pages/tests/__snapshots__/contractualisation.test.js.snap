// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | contractualisation render should match snapshot 1`] = `<contractualisation />`;
